packages/android_intent_plus/example/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
     path: ../
 
 dev_dependencies:
-  e2e: ^0.3.0
+  integration_test: ^1.0.2+2
```

Comment:
e2e is deprecated and replaced by integration_test.

Unfortunately `integration_test` does not support null safety yet!